src/app/lib/models/stream_info.js

```diff
@@ -76,8 +77,8 @@
 
             this.set('pieces', 0);
             this.set('downloaded', downloaded);
-            this.set('active_peers', info.numPeers);
-            this.set('total_peers', info.numPeers);
+            this.set('active_peers', torrent.numPeers);
```

Comment:
We can use the property _[numConns](https://github.com/feross/webtorrent/blob/master/lib/torrent.js#L200) to get the active peers here.
```javascript
this.set('active_peers', torrent._numConns);`
```

and add an extra info like the queued clients with the _[numQueued](https://github.com/feross/webtorrent/blob/master/lib/torrent.js#L194) property,

```javascript
this.set('queued_peers', torrent._numQueued);
```